### DISCUSSION TITLE
withResponsiveImagesIf() on conversions

### DISCUSSION THREAD

**Discussion by rcerljenko** [spatie/laravel-medialibrary#2083] [Ideas] [answered]
Can we also have newly added `withResponsiveImagesIf()` method on conversions? Eg:
```
$this->addMediaConversion('thumb')->withResponsiveImagesIf(/* some condition */);
```

**Comment by freekmurze** [comment]
If the implementation is simple, I'd accept a PR with tests and an updated readme. I'll only merge if this can be solved in an easy way.

**Comment by rcerljenko** [comment, marked as answer]
here you go: https://github.com/spatie/laravel-medialibrary/pull/2086
